torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13954,6 +14034,12 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                #               Expected a value of type 'List[Tensor]' for argument
                #               'tensors' but instead found type 'Tensor (inferred)'.
                DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_jit_alias_remapping'),
+               # RuntimeError: Batching rule not implemented for aten::concatenate
+               DecorateInfo(unittest.expectedFailure, 'TestVmapOperatorsOpInfo',
```

Comment:
Failures for functorch are defined a bit differently.

See for ref: https://github.com/pytorch/pytorch/blob/6a3ecda5a25025d48bbc5f0215db8c338745ef79/functorch/test/test_vmap.py#L3426-L3432